src/unix/notbsd/linux/mod.rs

```diff
@@ -407,7 +409,9 @@ extern {
     pub fn tmpfile64() -> *mut ::FILE;
     pub fn fgetpos64(stream: *mut ::FILE, ptr: *mut fpos64_t) -> ::c_int;
     pub fn fsetpos64(stream: *mut ::FILE, ptr: *const fpos64_t) -> ::c_int;
+    #[cfg(not(any(target_arch = "powerpc64", target_arch = "powerpc64le")))]
     pub fn fstat64(fildes: ::c_int, buf: *mut stat64) -> ::c_int;
```

Comment:
Are these functions not present in the powerpc libc at all? Or are they just present in future versions?